misc/performance.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import absolute_import
+from __future__ import print_function
+# This file is part of BurnMan - a thermoelastic and thermodynamic toolkit for the Earth and Planetary Sciences
+# Copyright (C) 2012 - 2015 by the BurnMan team, released under the GNU GPL v2 or later.
+
+import os, sys, numpy as np, matplotlib.pyplot as plt
+#hack to allow scripts to be placed in subdirectories next to burnman:
+if not os.path.exists('burnman') and os.path.exists('../burnman'):
+    sys.path.insert(1,os.path.abspath('..'))
+
+import burnman
+from burnman import minerals
+
+
+
+def test_grueneisen():
+    for aa in range(0,500):
+        a = burnman.eos.SLB3()
+        m = minerals.SLB_2011.fayalite()
+        x = 0
+        for i in range(0,10000):
+            x += a.grueneisen_parameter(1e9,1e4,1.1,m.params)
+    return x
+
+import timeit
+
+if False:
+    r=timeit.timeit('__main__.test_grueneisen()', setup="import __main__", number=3)
+    print("test_grueneisen", r)
```

Comment:
I used this for testing if my changes had a positive impact. I agree it is not great to have it in here as is. I will update it.